src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,102 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serving multiple applications including James. 
+The keyspace of each James server need to be protected from another, 
+that also means, by the credential provided for a James server, 
+I cannot use it to access or modify other keyspaces. 
+
+We recommend you to not use the default Cassandra credential and provide 
+a different one with a minimized permissions for each application. 
+
+### Prerequisites
+
+We're gonna use the super user to create roles and grant permissions for them. 
+To do that, Cassandra requires you to login via username/password authentication 
+and enable granting in cassandra configuration file.
+
+For example:
+```
+echo -e "\nauthenticator: PasswordAuthenticator" >> /etc/cassandra/cassandra.yaml
+echo -e "\nauthorizer: org.apache.cassandra.auth.CassandraAuthorizer" >> /etc/cassandra/cassandra.yaml
+```
+
+### Prepare Cassandra roles & keyspaces for James  
+
+#### Create a role
+
+Visit [cassandra documentation](http://cassandra.apache.org/doc/latest/cql/security.html) section `CREATE ROLE` for more information
+
+E.g.
+```
+CREATE ROLE linagora WITH PASSWORD = 'linagora' AND LOGIN = true;
+```
+#### Create a keyspace
+
+Visit [cassandra documentation](http://cassandra.apache.org/doc/latest/cql/ddl.html) section `CREATE KEYSPACE` for more information
+
+E.g.
+```
+CREATE KEYSPACE linagora_keyspace WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1};
```

Comment:
I really dislike using examples that are not suited for production.
People copy/paste things in a rush just to discover that they actually configured replication_factor to 1 once they lost all their data

Reply:
I'll remove it right away, I'm not sure I can provide an ideal key space creation for every users.